docs/pbc/all/tax-management/202307.0/vertex/integrate-vertex.md

```diff
@@ -0,0 +1,520 @@
+---
+title: Integrate Vertex
+description: Find out how you can integrate Vertex into your Spryker shop
+draft: true
+last_updated: Aug 3, 2023
+template: howto-guide-template
+---
+
+## Step 1 - Integrate the Tax App module
+
+To enable the Tax app integration, use the [spryker/tax-app](https://github.com/spryker/tax-app) module.
+
+The Tax app requires the following Spryker modules:
+
+  *  `"spryker/calculation-extension": "^1.0.0"`
+  *  `"spryker/event": "^2.1.0"`
+  *  `"spryker/event-behavior": "^1.23.0"`
+  *  `"spryker/kernel": "^3.30.0"`
+  *  `"spryker/log": "^3.0.0"`
+  *  `"spryker/message-broker": "^1.5.0"`
+  *  `"spryker/message-broker-extension": "^1.0.0"`
+  *  `"spryker/propel-orm": "^1.0.0"`
+  *  `"spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0"`
+  *  `"spryker/store": "^1.16.0"`
+  *  `"spryker/tax-app-extension": "^0.1.0"`
+  *  `"spryker/transfer": "^3.27.0"`
```

Comment:
if something is part of the default Spryker setup, let's add only the required packages (or their minimal versions) which are missing in the latest Demoshop (202307.0)